script.lua

```diff
@@ -2273,6 +2273,28 @@ function isObjectInHand(obj)
     return false
 end
 
+function removeButtons(params)
```

Comment:
are you imagine the impending code would also be calling `removeButtons` which is why you're passing in function_owner for example instead of just always assuming it's self

Reply:
I'm definitely imagining that the impending code would be calling `removeButtons`, though that's not quite why I'm passing `function_owner`.

`removeButtons()` removes *all* buttons that match all the passed parameters. So if the impending code (or another piece of custom code) added buttons that called a function named  `grabReminder`, `noFear`, `noHeal`, or the like on a *different* object, those would still get accidentally removed. I figure it's *unlikely* that anything would add other buttons calling functions of the same name on different objects, but by specifying both `click_function` and `function_owner`, it's *sure* to uniquely identify the function that's being called.

Sure, it could still remove multiple buttons if they call the same function on the same object. But at least in the case of `grabReminder`, it's *appropriate* to remove all buttons that call the same function, as that function errors if called more than once